app/Http/Controllers/Migration/StepsController.php

```diff
@@ -609,4 +609,35 @@ private function getCredits()
 
         return $transformed;
     }
+
+    private function getDocuments()
+    {
+        $documents = Document::where('account_id', $this->account->id)->get();
+
+        $transformed = [];
+
+        // Notes: 
+        // Missing: public_id
+
+        foreach ($documents as $document) {
```

Comment:
In V1 we also have invoice_id and expense_id to bring across.

in V2, we use polymorphic types, we'll then be able to attach the documents to the entities over there.